01-hello-world/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>LocalStack S3 website</title>
+    <meta charset="UTF-8">
+    <style>
+        body {
+            font-family: sans-serif;
+            color: #ffffff;
+            background-color: #2A1864;
+        }
+    </style>
+</head>
+<body>
+    <h1>You are running LocalStack 🎉</h1>
+    <p>
+        Welcome to the EuroPython 2023 LocalStack workshop!
```

Comment:
Nit: Let's make sure we update this before the talk (can be in a follow-up, not this PR)